tests/src/rules/no-extraneous-dependencies.js

```diff
@@ -380,74 +378,46 @@ ruleTester.run('no-extraneous-dependencies', rule, {
   ],
 });
 
-// TODO: figure out why these tests fail in eslint 4
-describe('TypeScript', { skip: semver.satisfies(eslintPkg.version, '^4') }, function () {
-  getTSParsers().forEach((parser) => {
-    const parserConfig = {
-      parser,
-      settings: {
-        'import/parsers': { [parser]: ['.ts'] },
-        'import/resolver': { 'eslint-import-resolver-typescript': true },
```

Comment:
should this maybe be kept, but add `node: true`?

Reply:
Yes, good idea. Done.